_posts/2013-09-26-authentication.markdown

```diff
@@ -10,6 +10,10 @@ title: Authentication Example
 
 Due to the sensitive nature of payroll, all potential integrations must be vetted and approved by Gusto.
 
+When developing an integration, all API calls should be made to api.gusto-demo.com. Once the integration is reviewed by Gusto, we’ll request a callback URI for your production system and issue production API keys at which point all calls should be made to api.gusto.com.
```

Comment:
I would suggest a couple of changes for readability:

- Put backticks (`) around api.gusto-demo.com and api.gusto.com to make them stand out.
- Break up the second sentence into two: `...issue production API keys. After you receive production API keys, all calls should be made to api.gusto.com.`

Reply:
Thank you, Dallas!